app/components/RequestPreview.tsx

```diff
@@ -171,6 +174,14 @@ function RequestPreview({ children, request, teams = [] }: Props) {
             </tr>
           )}
           <FormattedList list={fullIdpDisplay} title="Identity Providers Required:" inline testid="idp-required" />
+          {hasBCSC && (
+            <tr>
+              <td>Privacy Zone:</td>
+              <td>
+                <SemiBold>{privacyZone || 'Unavailable'}</SemiBold>
```

Comment:
@jlangy if privacy zone is unavailable then we could always show the hard coded ones ryt?

Reply:
Oh yeah, like fallback first to the ones in the constants file? I can add that in